git_theta/git_utils.py

```diff
@@ -193,74 +196,45 @@ def remove_file(f, repo):
         repo.git.rm(f)
 
 
-def git_lfs_install():
-    """
-    Run the `git lfs install` command
+def get_file_version(repo, path, commit_hash):
+    path = get_relative_path_from_root(repo, path)
+    try:
+        tree = repo.commit(commit_hash).tree
+        if path in tree:
+            return tree[path]
+        else:
+            return None
+    except git.BadName:
+        return None
 
-    Returns
-    -------
-    int
-        Return code of `git lfs install`
-    """
-    out = subprocess.run(["git", "lfs", "install"])
-    return out.returncode
 
+def get_head(repo):
+    try:
+        head = repo.commit("HEAD")
+        return head.hexsha
+    except git.BadName:
+        None
```

Comment:
Probably do an explicit `return None` instead of just `None`, in this case `None` is returned but that is just because it is the default value if there are no returns.

Reply:
Lol whoops, fixed